inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -842,23 +900,34 @@ obs_start <- week2date(sprintf("%s-1", first_week))
 
 # return last day of reporting week 
 obs_end   <- week2date(sprintf("%s-7", reporting_week))
```

Comment:
A couple of things I'm thinking about:

1. sprintf is a weirdly-named function and we should probably switch to `glue::glue()`
2. We can pre-define the observation end ([as above](https://github.com/R4EPI/sitrep/pull/172/files#r319977197)) to avoid this sprintf business
3. This chunk is `eval = FALSE`, which means that we should comment these lines out.


But, if we pre-define `obs_end` above, then we can rewrite this chunk to look like this:
```r
linelist_cleaned <- linelist_cleaned %>% 
  filter(date_of_consultation_admission <= as.Date(obs_end))

# define the first week of outbreak (date of first case)
first_week <- min(linelist_cleaned$epiweek)

# outbreak start 
# return the first day in the week of first case 
obs_start <- as.Date(first_week)

```

Reply:
Used str_glue() since we already have that package loaded.